src/app/withdraw/[txid]/components/withdrawal-status.tsx

```diff
@@ -86,6 +87,15 @@ function Content(initialData: Props) {
     }
   }, [stepper, status]);
 
+  useEffect(() => {
+    if (
+      status === WithdrawalStatus.confirmed ||
+      status === WithdrawalStatus.failed
```

Comment:
nit/fyi: we don't need the `failed` case as the "balance" we query for with `get-balance` includes both the unlocked and the locked amount, so in case of a failed withdrawal we would just unlock the locked amount (but the total will remain the same). But this is cleaner, so let's keep it as is.

Reply:
fair point